scripts/train.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+from datetime import datetime
+
+from arcsf import ExperimentConfig, load_model_and_tokenizer, load_trainer
+from arcsf.data.tofu import load_tofu
+from arcsf.utils import seed_everything
+
+
+def main(experiment_name):
+    start_time = datetime.strftime(datetime.now(), "%Y%m%d-%H%M%S-%f")
+
+    # Step 1: Process configs to dicts
+    experiment_config = ExperimentConfig.from_yaml(
+        f"configs/experiment/{experiment_name}.yaml"
```

Comment:
See other comment re making assumptions about what directory the script has been called from

Reply:
as above